m5paperwardriver/m5paperwardriver.ino

```diff
@@ -155,26 +178,31 @@ void loop() {
     Serial.println("Waiting for valid GPS data...");
   }
 
-  int n = WiFi.scanNetworks();
+  int n = WiFi.scanNetworks(false,true,false,110); // Scan hidden networks at 110ms per channel
```

Comment:
most things (such as kismet) do 200ms by default.  110ms is enough, but 200ms would notably increase your chances as you can lose 1 packet (50%) and still see the AP